apps/docs/pages/guides/resources/migrating-to-supabase/mysql.mdx

```diff
@@ -0,0 +1,86 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  title: 'Migrate Your MySQL Database to Supabase',
+  description: 'Migrate your MySQL database to Supabase Postgres database.',
+}
+
+This guide aims to exhibit the process of transferring your MySQL database to Supabase's Postgres database. Supabase is a robust and open-source platform offering a wide range of backend features, including a PostgreSQL database, authentication, instant APIs, edge functions, real-time subscriptions, and storage. Migrating your MySQL database to Supabase's PostgreSQL enables you to leverage PostgreSQL's capabilities and access all the features you need for your project.
+
+## Retrieve Your MySQL Database Credentials
+
+Before you begin the migration, you need to collect essential information about your MySQL database. Follow these steps:
+
+1. Log in to your MySQL database provider.
+
+1. Locate and note the following database details:
+   - Hostname or IP address
+   - Database name
+   - Username
+   - Password
+
+## Retrieve your Supabase Host [#retrieve-supabase-host]
+
+1. If you're new to Supabase, [create a project](https://supabase.com/dashboard).
+1. Go to the [Database settings](https://supabase.com/dashboard/project/_/settings/database) for your project in the Supabase Dashboard.
+1. Under **Connection Info**, note your Host (`$SUPABASE_HOST`).
+1. Save your password or [reset it](https://supabase.com/dashboard/project/_/settings/database) if you forgot it.
+
+![Finding Supabase host address](/docs/img/guides/resources/migrating-to-supabase/amazon-rds/supabase_dashboard.png)
+
+## Migrate the database
+
+The fastest way to migrate your database is with the Supabase migration tool on
+[Google Colab](https://colab.research.google.com/github/mansueli/Supa-Migrate/blob/main/Amazon_RDS_to_Supabase.ipynb).
+
+Alternatively, you can use [pgloader](https://github.com/dimitri/pgloader), a flexible and powerful data migration tool that supports a wide range of source database engines, including MySQL and MS SQL, and migrates the data to a Postgres database. For databases using the Postgres engine, we recommend using the pg_dump and psql command line tools, which are included in a full PostgreSQL installation.
```

Comment:
Noticing some guides have `pg_dump` and `psql` linked, and some don't. Could we keep it consistent across each guide?